src/features/runtime-checks.js

```diff
@@ -729,7 +729,7 @@ export default class RuntimeChecks extends ContentFeature {
 
     overloadDateGetTimezoneOffset (config) {
         const offset = (new Date()).getTimezoneOffset()
-        defineProperty(globalThis.Date.prototype, 'getTimezoneOffset', {
+        this.defineProperty(globalThis.Date.prototype, 'getTimezoneOffset', {
```

Comment:
@jonathanKingston not sure if Messaging will actually work in injected scripts. The tests pass, but how do I test it manually to verify?

Reply:
I doubt it will work, I don't think we wired it in.
We have a test page that you can run these locally: https://github.com/duckduckgo/content-scope-scripts/blob/main/integration-test/test-pages/runtime-checks/pages/basic-run.html with the config: https://github.com/duckduckgo/content-scope-scripts/blob/main/integration-test/test-pages/runtime-checks/config/basic-run.json

Reply:
Ok, I just confirmed that the messages are silently ignored, so we can keep using the wrapper methods, but debug flags won't be delivered from runtime checks at the moment. Which is fine for now.

Reply:
Thanks for checking :)